src/views/SluggoNavbar/__tests__/navbar.test.ts

```diff
@@ -0,0 +1,152 @@
+import { fireEvent, render, waitFor } from "@testing-library/vue";
+import "@testing-library/jest-dom";
+import UserProvidedNavbar from "@/views/SluggoNavbar/UserProvidedNavbar.vue";
+import {
+  mockRouter,
+  mockUser,
+  sampleTeamData
+} from "@/views/SluggoNavbar/__mocks__/navbarDataMock";
+import * as TeamAPI from "@/api/teams";
+import * as AuthAPI from "@/api/auth";
+
+const renderNavbarAndAwaitRouter = async () => {
+  const renderResult = render(UserProvidedNavbar, {
+    global: {
+      plugins: [mockRouter]
+    }
+  });
+
+  await mockRouter.isReady();
```

Comment:
Sam says to add a comment that this needs to be done to avoid throwing errors on incomplete rendered component.